### PR TITLE
Add 1.28 CloudStack E2E

### PR DESCRIPTION
**Pull request by tatlat**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/cloudstack_test.go [hunk 1]
```diff
@@ -1659,6 +1952,46 @@ func TestCloudStackUpgradeKubernetes127MulticlusterWorkloadClusterWithGithubFlux
...
+func TestCloudStackUpgradeKubernetes128MulticlusterWorkloadClusterWithGithubFlux(t *testing.T) {
+	provider := framework.NewCloudStack(t, framework.WithCloudStackRedhat126())
+	test := framework.NewMulticlusterE2ETest(
+		t,
+		framework.NewClusterE2ETest(
+			t,
+			provider,
+			framework.WithFluxGithub(),
+			framework.WithClusterFiller(
+				api.WithKubernetesVersion(v1alpha1.Kube126),
```
Comment (mitalipaygude) [inline test/e2e/cloudstack_test.go:1964]: should this be 127?

File: test/e2e/cloudstack_test.go [hunk 2]
```diff
@@ -1659,6 +1952,46 @@ func TestCloudStackUpgradeKubernetes127MulticlusterWorkloadClusterWithGithubFlux
...
+				api.WithWorkerNodeCount(1),
+				api.WithStackedEtcdTopology(),
+			),
+		),
+		framework.NewClusterE2ETest(
+			t,
+			provider,
+			framework.WithFluxGithub(),
+			framework.WithClusterFiller(
+				api.WithKubernetesVersion(v1alpha1.Kube126),
```
Comment (mitalipaygude) [inline test/e2e/cloudstack_test.go:1975]: should this be 127?

File: test/e2e/cloudstack_test.go (hunk 1 above)
Comment (tatlat) [reply test/e2e/cloudstack_test.go]: yes

**Review by mitalipaygude — APPROVED** [review]

**Pull request merged**